train_segmentation_model.py

```diff
@@ -1,7 +1,11 @@
 import shutil
 import os
 import random
+random.seed(1)
 import numpy as np
+np.random.seed(12)
+from tensorflow import set_random_seed
+set_random_seed(123)
```

Comment:
same comment here that I left on infer (although these should probably be in the config file instead)